x-pack/plugins/security_solution/server/endpoint/routes/resolver/utils/fetch.ts

```diff
@@ -249,14 +255,19 @@ export class Fetcher {
    * @param limit the upper bound number of alerts to return
    * @param after a cursor to use as the starting point for retrieving alerts
    */
-  public async alerts(limit: number, after?: string): Promise<ResolverRelatedAlerts> {
-    const alertsHandler = new RelatedAlertsQueryHandler(
+  public async alerts(
```

Comment:
:grey_question: The comments here could be improved: Add the new param (filter) to the param list above and add some explanation of how they work together. (Is the limit calculated before or after the filter is applied? etc.)

Reply:
Thanks for catching that. I'll update.